daemon/daemon_unix.go

```diff
@@ -603,15 +603,7 @@ func configureMaxThreads(config *Config) error {
 // configureKernelSecuritySupport configures and validates security support for the kernel
 func configureKernelSecuritySupport(config *Config, driverName string) error {
 	if config.EnableSelinuxSupport {
-		if selinuxEnabled() {
-			// As Docker on overlayFS and SELinux are incompatible at present, error on overlayfs being enabled
-			if driverName == "overlay" {
-				return fmt.Errorf("SELinux is not supported with the %s graph driver", driverName)
-			}
-			logrus.Debug("SELinux enabled successfully")
-		} else {
-			logrus.Warn("Docker could not enable SELinux on the host system")
-		}
+		logrus.Warn("Docker could not enable SELinux on the host system")
```

Comment:
Also, if this depends on kernel 4.9, should we use the old behavior if an older kernel is in use? (or is there another way to detect if it's supported?)